spec/factories/recipes.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :recipe do
+    title { 'MyString' }
+    ingredients { 'MyText' }
+    instructions { 'MyText' }
+    references { '' }
```

Comment:
We've found that the basic factory definition -- the one named after the model -- is easiest to work with if it only defines the minimum required values. In this case that's `user`.

If I were working on this I'd make sure the `recipe` factory definition creates a `user`, and nothing else.

Reply:
@orhanna14 if you're curious about some of the _why_ behind this advice, you might find these two articles from thoughtbot blog helpful:

1. https://thoughtbot.com/blog/why-factories
2. https://thoughtbot.com/blog/factories-should-be-the-bare-minimum